### PR TITLE
Cmvp 226 header component

### PR DESCRIPTION
**Pull request by JEduardoMP**
Created the following sections:
- About me
- Goals
- Open to..
Created the function to calculate the completed percentage in personal profile.
Created component of feedback to saving changes in personal profile

### CODE REVIEW

File: src/components/molecules/AboutMe/index.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import * as S from '../Goals/styles'
+
+export const AboutMe = ({ setData, data }) => (
+  <S.GoalsContainer>
+    <input
+      type="text"
+      placeholder="About me"
+      value={data.aboutMe}
+      onChange={e => setData(prev => ({ ...prev, aboutMe: e.target.value }))}
```
Comment (fercc1097) [inline src/components/molecules/AboutMe/index.jsx:12]: Please use formik to handle onChange and Value

File: src/components/molecules/FormProfile/index.jsx [hunk 2]
```diff
@@ -2,69 +2,113 @@ import { useState } from 'react'
...
+    if (e.target.checked) {
+      setData(prev => ({
+        ...prev,
+        whoDoYouWantToConnect: [...prev.whoDoYouWantToConnect, e.target.value],
+      }))
+    } else {
+      setData(prev => ({
+        ...prev,
+        whoDoYouWantToConnect: [...prev.whoDoYouWantToConnect].filter(
+          el => el !== e.target.value,
```
Comment (fercc1097) [inline src/components/molecules/FormProfile/index.jsx:28]: what is this el? 
e = event 
el = ?

File: src/components/molecules/FormProfile/index.jsx [hunk 3]
```diff
@@ -2,69 +2,113 @@ import { useState } from 'react'
...
+            </>
+          ))}
 
-        <div className="own-answers">
-          <input type="text" placeholder="Add your own answer" />
-          <Button type="text" onClick={addNewAnswer}>
-            Add
-          </Button>
+          <div className="own-answers">
+            <input type="text" placeholder="Add your own answer" />
```
Comment (fercc1097) [inline src/components/molecules/FormProfile/index.jsx:104]: Please Use Material UI and formik

File: src/components/molecules/FormProfile/styles.jsx [hunk 4]
```diff
@@ -112,3 +113,15 @@ export const FormProfileContainer = styled.div`
     }
   }
 `
+
+export const ModalCustomStyled = styled(Box)`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 400;
```
Comment (fercc1097) [inline src/components/molecules/FormProfile/styles.jsx:122]: 400 miss the units, em, rem, px, etc..

File: src/components/molecules/FormProfile/styles.jsx [hunk 5]
```diff
@@ -112,3 +113,15 @@ export const FormProfileContainer = styled.div`
...
+  background-color: red;
```
Comment (fercc1097) [inline src/components/molecules/FormProfile/styles.jsx:123]: Please use colors listed on theme/colors or add a new one

File: src/components/molecules/Goals/index.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+      goals: {
+        checked: checked,
+        value: prev.goals.value,
+      },
+    }))
+  }
+
+  return (
+    <S.GoalsContainer>
+      <input
```
Comment (fercc1097) [inline src/components/molecules/Goals/index.jsx:21]: The same, please use formik

Comment (fercc1097) [reply src/components/molecules/Goals/index.jsx:21]: And MUI Inputs

File: src/components/molecules/OpenTo/index.jsx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+            <TextField
+              helperText="Maximum 15 words"
+              label="Your own answer"
+              color="secondary"
+            />
+          </div>
+        </Modal>
+      )}
+      <S.OpenToContainer>
+        <h4>Open to...</h4>
```
Comment (fercc1097) [inline src/components/molecules/OpenTo/index.jsx:45]: We have a Text components which may be easier to use

**Review by fercc1097 — CHANGES_REQUESTED** [review]
Please use MUI and formik, also take in mind the Text component and colors in theme folder

**Review by Moibe — APPROVED** [review]

**Review by cochairdev — APPROVED** [review]

File: src/components/molecules/FormProfile/index.jsx [hunk 8]
```diff
@@ -2,69 +2,113 @@ import { useState } from 'react'
...
+              color="secondary"
+            />
+          </S.ModalCustomStyled>
+        </Modal>
+      )}
+      <S.FormProfileContainer>
+        <div className="title-container">
+          <div className="title">
+            <AutoAwesomeIcon />
+            <h3>
```
Comment (fercc1097) [inline src/components/molecules/FormProfile/index.jsx:60]: Please use here the Text component and MUI for inputs please

File: src/components/molecules/OpenTo/index.jsx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+        openTo: [...prev.openTo].filter(el => el !== e.target.value),
+      }))
+    }
+  }
+  return (
+    <>
+      {openModal && (
+        <Modal open onClose={() => setOpenModal(false)}>
+          <div>
+            <h2>Who do you want to connect with?</h2>
```
Comment (fercc1097) [inline src/components/molecules/OpenTo/index.jsx:36]: The same here with Text Component

File: src/components/molecules/OpenTo/styles.jsx [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+  padding: 0 !important;
+  background-color: transparent !important;
+  border: none !important;
+  h4 {
+    font-size: 1rem;
+  }
+  p {
+    font-size: 0.9rem;
+    letter-spacing: -0.1px;
+    color: #666666;
```
Comment (fercc1097) [inline src/components/molecules/OpenTo/styles.jsx:15]: Please add this color too

File: src/components/molecules/Progress/index.jsx [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+import { CustomProgress } from '@/components/atoms'
+
+export const Progress = ({ percentage, changeDetected }) => (
+  <S.ProgressContainer>
+    <div>
+      {changeDetected === 'saving' && (
+        <S.Saving>
+          <CircularProgress />
+          <div className="text-container">
+            <p className="saving">Saving</p>
```
Comment (fercc1097) [inline src/components/molecules/Progress/index.jsx:15]: Use Text component in this component too

**Review by fercc1097 — CHANGES_REQUESTED** [review]

File: src/components/molecules/Progress/index.jsx [hunk 12]
```diff
@@ -11,26 +11,32 @@ export const Progress = ({ percentage, changeDetected }) => (
       {changeDetected === 'saving' && (
         <S.Saving>
           <CircularProgress />
-          <div className="text-container">
-            <p className="saving">Saving</p>
-            <p>your changes</p>
-          </div>
+          <S.TextContainer>
+            <Typography variant="body1" className="saving">
```
Comment (fercc1097) [inline src/components/molecules/Progress/index.jsx:15]: Please Use molecules/Text component instead of Typography

**Review by fercc1097 — APPROVED** [review]

**Pull request merged**